### PR TITLE
Created UpdatePassword component

### LINKED ISSUE #10: Update Password
- Create a form with the old password, new password and confirm password fields.
- Validate that the new password matching the password format.
- Validate that password and confirm password fields are matched.
- Submit the update password form.
- Handle displaying any errors from the backend response.
- Handle displaying success message if request success.

### PR DESCRIPTION
**Pull request by mohamedsgap**
Just finished creating `UpdatePassword` component #10 , with Test Specs!

Screenshot from 2020-09-07 02-56-03

Screenshot from 2020-09-07 02-59-49

### CODE REVIEW

File: src/containers/UpdatePassword/UpdatePassword.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+  const updateNewPassword = (e) => {
+    setNewPassword(e.target.value);
+  };
+  const updateConfirmNewPassword = (e) => {
+    setConfirmNewPassword(e.target.value);
+  };
+
+  const checkValues = () => {
+    if (newPassword === "" || confirmNewPassword === "" || oldPassword === "") {
+      setError("CAN NOT BE EMPTY FIELD!");
```
Comment (AbdullahMuhammed5) [inline src/containers/UpdatePassword/UpdatePassword.js:72]: Great work, I just thinking that you should not make error messages all caps like these.

**Review by AbdullahMuhammed5 — DISMISSED** [review]

File: src/containers/UpdatePassword/UpdatePassword.js (hunk 1 above)
Comment (mohamedsgap) [reply src/containers/UpdatePassword/UpdatePassword.js:72]: @AbdullahMuhammed5 I think, it's for UX purposes, just to be more noticed by user attention! if the other fellows don't like it, we can revert it back! when we combine all `USER SIGNUP/LOGIN` components together :+1:

**Review by aashrafh — DISMISSED** [review]

**Comment by mohamedsgap** [comment]
hey @mwissem @aashrafh  @AbdullahMuhammed5 , I need ur review on this PR after resolving a conflict in `gitignore`  file

**Review by AbdullahMuhammed5 — APPROVED** [review]

File: src/containers/UpdatePassword/__snapshots__/UpdatePassword.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,91 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (aashrafh) [inline src/containers/UpdatePassword/__snapshots__/UpdatePassword.test.js.snap:1]: Add snapshot test files to ```gitignore``` file

**Review by aashrafh — APPROVED** [review]

File: src/containers/UpdatePassword/__snapshots__/UpdatePassword.test.js.snap (hunk 2 above)
Comment (mohamedsgap) [reply src/containers/UpdatePassword/__snapshots__/UpdatePassword.test.js.snap:1]: I don't think so, We gonna take about the unit testing approach in our meeting

**Comment by mohamedsgap** [comment]
hey @RanaEmad, I think this PR can be merged too!

**Comment by RanaEmad** [comment]
> hey @RanaEmad, I think this PR can be merged too!

Hi @mohamedsgap, has @mwissem reviewed this before you resolved the conflicts?

**Comment by mohamedsgap** [comment]
he told that he gonna review, but i think he missed, so  you can review and merge it anyway @RanaEmad

**Review by ghost — APPROVED** [review]

**Pull request merged**